bpf/process/bpf_fork.c

```diff
@@ -65,10 +64,12 @@ BPF_KPROBE(event_wake_up_new_task, struct task_struct *task)
 	 * before the execve hook point if they changed or not.
 	 * This needs to be converted later to credentials.
 	 */
-	get_current_subj_caps(&caps, task);
-	curr->caps.permitted = caps.permitted;
-	curr->caps.effective = caps.effective;
-	curr->caps.inheritable = caps.inheritable;
```

Comment:
Unrelated to that PR, but these assignments seems to be redundant as we did `get_current_subj_caps`? And this is the reason that you removed those now, right?

Reply:
This just minor optimization, since old code was reading in new var caps then copying back to curr->caps.$field, see next line in the patch, now we read directly into curr->caps , so just saved some cycles.